### PR TITLE
API Issue #62: Fix numbering in 'hook_update_N()' docblock.

### LINKED ISSUE #62: Wrong description about function hook_update_N
[Original page](https://api.backdropcms.org/api/backdrop/core%21modules%21system%21system.api.php/function/hook_update_N/1)

There are three wrong version number:
> **mymodule_update_1200()**: This is the first update to get the database ready to run mymodule 7.x-2.\*. Users can directly update from 1.x-2.* to 2.x-2.* and they get all 10xx and 12xx updates, but not 11xx updates, because those reside in the 1.x-1.x branch only.

Fixed:
**mymodule_update_1200()**: This is the first update to get the database ready to run mymodule 1.x-2.\*. Users can directly update from 7.x-2.* to 1.x-2.* and they get all 10xx and 12xx updates, but not 11xx updates, because those reside in the 1.x-1.x branch only.

Related: [D7 function hook_update_N](https://api.drupal.org/api/drupal/modules%21system%21system.api.php/function/hook_update_N/7.x)

### PR DESCRIPTION
**Pull request by ghost**
Fixes https://github.com/backdrop-ops/api.backdropcms.org/issues/62

### CODE REVIEW

File: core/modules/system/system.api.php [hunk 1]
```diff
@@ -2801,9 +2801,9 @@ function hook_install() {
  * - mymodule_update_1100(): This is the first update to get the database ready
  *   to run mymodule 1.x-1.*.
  * - mymodule_update_1200(): This is the first update to get the database ready
- *   to run mymodule 7.x-2.*. Users can directly update from 1.x-2.* to 2.x-2.*
- *   and they get all 10xx and 12xx updates, but not 11xx updates, because
- *   those reside in the 1.x-1.x branch only.
+ *   to run mymodule 1.x-2.*. Users can directly update from Drupal 7.x to
+ *   1.x-2.* and they get all 10xx and 12xx updates, but not 11xx updates
```
Comment (klonos) [inline core/modules/system/system.api.php]:
```suggestion
 *   Backdrop 1.x-2.*, and they get all the 10xx and 12xx updates, but not the
```

File: core/modules/system/system.api.php [hunk 2]
```diff
@@ -2801,9 +2801,9 @@ function hook_install() {
...
+ *   because those reside in the 1.x-1.x branch only.
```
Comment (klonos) [inline core/modules/system/system.api.php]:
```suggestion
 *   11xx updates, because those reside in the 1.x-1.x branch only.
```

**Comment by ghost** [comment]
Thanks for the suggestions @klonos!

**Comment by ghost** [comment]
Merged into 1.x and 1.17.x.

**Pull request merged**